src/components/calcite-list/calcite-list.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, h, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+
+/**
+ * A general purpose list that enables users to construct list items that conform to Calcite styling.
+ * @slot - todo document.
+ */
+@Component({
+  tag: "calcite-list",
+  styleUrl: "calcite-list.scss",
+  shadow: true
+})
+export class CalciteList {
```

Comment:
Can we make the list manage tabbing among its items when navigating via keyboard? This would help with accessibility and help keyboard users easily tab through the list without having to go through all items.

Reply:
I dont think the list role would do that.

Reply:
Right, this goes against my comment to use that role. I guess my suggestion is based on how close this and the other lists are in name and me assuming they would behave similarly. Clear doc and renaming pick/value list would help others avoid my assumption, right?

Reply:
Yes, I think we can evaluate pick-list and value-list at a later time. They should probably conform to more useful names that are common in UI libraries.

Reply:
Sounds good. That should help clear things up for sure. Until then, clear doc would be the key.